Storage/SQL/BrowserDB.swift

```diff
@@ -92,6 +92,13 @@ open class BrowserDB {
             log.debug("Creating db: \(file) with secret = \(secretKey!)")
         }
 
+        if !leaveClosed {
+            self.firstOpen()
+        }
+    }
+
+    // Do the same work that we normally do at the end of init.
+    public func firstOpen() {
```

Comment:
nit: maybe a better name would be `updateSchema()` or something along those lines?